lib/solaar/__init__.py

```diff
@@ -19,6 +19,7 @@
 import sys as _sys
 
 NAME = "solaar"
+DISPLAY_NAME = "Solaar"
```

Comment:
The display name should be derived from the name, not a separate constant.  Even better, use only Solaar and then .lower() where a lower-case name is needed.   There are already several places where .lower() is used.